onebusaway-android/src/main/java/org/onebusaway/android/ui/NavigationDrawerFragment.java

```diff
@@ -82,8 +82,6 @@
 
     protected static final int NAVDRAWER_ITEM_PLAN_TRIP = 6;
 
-    protected static final int NAVDRAWER_ITEM_POPULAR = 7;
-
```

Comment:
Note that popular still appears in the NAVDRAWER_TITLE_RES_ID, NAVDRAWER_ICON_RES_ID, and NAVDRAWER_ICON_SECONDARY_RES_ID arrays.  This is to avoid a bounds issue in makeNavDrawerItem.  We can remove this entry in these arrays by decrementing the other constants by 1.

Reply:
I think it's fine to leave the resource references as-is.  I'm also going to keep the constant here and add a comment explaining why it's still there with a reference to the issue number.